ui/apps/platform/src/Containers/Policies/Table/PoliciesTable.tsx

```diff
@@ -465,15 +471,37 @@ function PoliciesTable({
                 </Table>
             </PageSection>
             <ConfirmationModal
+                title={`${
+                    isExternalPolicySelected(policies, deletingIds)
+                        ? 'Delete policies?'
+                        : 'Permanently delete policies?'
+                } (${deletingIds.length})`}
                 ariaLabel="Confirm delete"
                 confirmText="Delete"
                 isLoading={isDeleting}
                 isOpen={deletingIds.length !== 0}
                 onConfirm={onConfirmDeletePolicy}
                 onCancel={onCancelDeletePolicy}
             >
-                Are you sure you want to delete {deletingIds.length}&nbsp;
-                {pluralize('policy', deletingIds.length)}?
+                {isExternalPolicySelected(policies, deletingIds) ? (
+                    <Flex direction={{ default: 'column' }}>
+                        <FlexItem>
+                            Deleted policies will be removed from the system and will no longer
+                            trigger violations.
+                        </FlexItem>
+                        <FlexItem>
+                            <em>
+                                The current selection includes at least one externally managed
+                                policy that will be restored automatically during the next resync.
```

Comment:
This sentence is both inconsistent with and less clear than the explanation in PolicyDetail.

Reply:
How about:

"The current selection includes one or more externally managed policies that will only be removed from the system temporarily. Locally managed policies will be removed permanently."